packages/components/atoms/f-form-field/README.md

```diff
@@ -14,91 +15,118 @@
 [![Coverage Status](https://coveralls.io/repos/github/justeat/f-form-field/badge.svg)](https://coveralls.io/github/justeat/f-form-field)
 [![Known Vulnerabilities](https://snyk.io/test/github/justeat/f-form-field/badge.svg?targetFile=package.json)](https://snyk.io/test/github/justeat/f-form-field?targetFile=package.json)
 
+---
 
 ## Usage
 
-1.  Install the module using NPM or Yarn:
+### Installation
 
-    ```bash
-    yarn add @justeat/f-form-field
-    ```
+Install the module using NPM or Yarn:
 
-    ```bash
-    npm install @justeat/f-form-field
-    ```
+```sh
+yarn add @justeat/f-form-field
+```
+
+```sh
+npm install @justeat/f-form-field
+```
 
-2.  Import the component
+### Vue Applications
 
-    You can import it in your Vue SFC like this (please note that styles have to be imported separately):
+You can import it in your Vue SFC like this (please note that styles have to be imported separately):
 
-    ```
-    import FormField from '@justeat/f-form-field';
-    import '@justeat/f-form-field/dist/f-form-field.css';
+```js
+import FormField from '@justeat/f-form-field';
+import '@justeat/f-form-field/dist/f-form-field.css';
 
-    export default {
-        components: {
-            FormField
-        }
+export default {
+    components: {
+        FormField
     }
-    ```
+}
+```
 
-    If you are using Webpack, you can import the component dynamically to separate the header bundle from the main `bundle.client.js`:
+If you are using Webpack, you can import the component dynamically to separate the `mega-modal` bundle from the main `bundle.client.js`:
 
-    ```
-    import '@justeat/f-form-field/dist/f-form-field.css';
+```js
+import '@justeat/f-form-field/dist/f-form-field.css';
 
-    export default {
-        components: {
-            ...
-            FormField: () => import(/* webpackChunkName: "form-field" */ '@justeat/f-form-field')
-        }
+export default {
+    components: {
+        // …
+        FormField: () => import(/* webpackChunkName: "form-field" */ '@justeat/f-form-field')
     }
+}
+```
+
+## Configuration
+
+### Props
+
+There may be props that allow you to customise its functionality.
+
+The props that can be defined are as follows (if any):
+
+| Prop  | Type  | Default | Description |
+| ----- | ----- | ------- | ----------- |
+
+### CSS Classes
```

Comment:
This section (CSS Classes) won't ever apply to form-field so it can be removed. Just for reference, we only expose classes in specific circumstances, so I think the modal is one of the only components we do this in.

Reply:
I left the empty table there because it shows it's not forgotten about, it's just empty.

Reply:
I don't think we'll use this in 90%* of the components (it's definitely not something we recommend doing, as we want to encapsulate the styles) – it's an edge case in the modal as they needed to style it in certain contexts.

* statistic made up